drift_dev/lib/src/cli/commands/make_migrations.dart

```diff
@@ -0,0 +1,505 @@
+  ${yellow.wrap("]")};
+
+  ${blue.wrap("final")} ${lightCyan.wrap("usersV2")} = ${yellow.wrap("[")}
+    v2.${green.wrap("User")}${magenta.wrap("(")}${lightCyan.wrap("id")}: ${green.wrap("Value")}${blue.wrap("(")}1${blue.wrap(")")}, ${lightCyan.wrap("name")}: ${green.wrap("Value")}${blue.wrap("(")}${lightRed.wrap("'Simon'")}${blue.wrap(")")}${magenta.wrap(")")},
+    v2.${green.wrap("User")}${magenta.wrap("(")}${lightCyan.wrap("id")}: ${green.wrap("Value")}${blue.wrap("(")}2${blue.wrap(")")}, ${lightCyan.wrap("name")}: ${green.wrap("Value")}${blue.wrap("(")}${lightRed.wrap("'John'")}${blue.wrap(")")}${magenta.wrap(")")},
+  ${yellow.wrap("]")};
+
+### Configuration
+
+This tool requires the database be defined in the build.yaml file.
+Example:
+
+${blue.wrap("""
+targets:
+  \$default:
+    builders:
+      drift_dev:
+        options:""")}
+          ${green.wrap("# Required: The name of the database and the path to the database file")}
+          ${blue.wrap("databases")}:
+            ${blue.wrap("my_database")}: ${lightRed.wrap("lib/database.dart")}
+
+            ${green.wrap("# Optional: Add more databases")}
+            ${blue.wrap("another_db")}: ${lightRed.wrap("lib/database2.dart")}
+
+
+          ${green.wrap("# Optional: The directory where the test files are stored")}: 
+          ${blue.wrap("test_dir")}: ${lightRed.wrap("test/drift/")} ${green.wrap("# (default)")}
+
+          ${green.wrap("# Optional: The directory where the schema files are stored")}:
+          ${blue.wrap("schema_dir")}: ${lightRed.wrap("drift_schemas/")}  ${green.wrap("# (default)")}
+
+""";
+
+  @override
+  String get name => "make-migrations";
+
+  @override
+  Future<void> run() async {
+    if (p.isAbsolute(cli.project.options.schemaDir)) {
+      cli.exit(
+          '`schema_dir` must be a relative path. Remove the leading slash');
+    }
+    if (p.isAbsolute(cli.project.options.testDir)) {
+      cli.exit('`test_dir` must be a relative path. Remove the leading slash');
+    }
+
+    /// The root directory where test files for all databases are stored
+    /// e.g /test/drift/
+    final rootSchemaDir = Directory(
+        p.join(cli.project.directory.path, cli.project.options.schemaDir))
+      ..createSync(recursive: true);
+
+    /// The root directory where schema files for all databases are stored
+    /// e.g /drift_schemas/
+    final rootTestDir = Directory(
+        p.join(cli.project.directory.path, cli.project.options.testDir))
+      ..createSync(recursive: true);
+
+    if (cli.project.options.databases.isEmpty) {
+      cli.logger.info(
+          'No databases found in the build.yaml file. Check here to see how to add a database TODO: ADD LINK');
+      exit(0);
```

Comment:
Don't exit with 0 here, it's not a successful command invocation. Use either 1 or 64 for usage errors.